terraform/environments/corporate-staff-rostering/ec2_common.tf

```
@@ -40,16 +40,45 @@ resource "aws_ssm_document" "cloud_watch_agent" {
   )
 }
 
-resource "aws_ssm_document" "ami_build" {
+resource "aws_ssm_document" "ami_build-command" {
```

Underline instead of dash.
```suggestion
resource "aws_ssm_document" "ami_build_command" {
```